client/packages/common/src/ui/components/inputs/Autocomplete/AutocompleteList.tsx

```diff
@@ -72,14 +74,28 @@ export const AutocompleteList = <T,>({
     mappedOptions = options;
   }
 
+  const handleChange: AutocompleteOnChange<T | T[]> = (
+    event,
+    value,
+    reason,
+    details
+  ) => {
+    onChange?.(event, value, reason, details);
+    const scrollPos = listboxRef.current?.scrollTop || 0;
+    window.setTimeout(
```

Comment:
yeah, well, it works.
The issue is that when you select, this component and its parent both re-render. Tried avoiding that, but it is tricky because the selected options need to change, and the selection state requires rendering.
This is a quick fix to get it working - just scroll to where you are after the render.

It looks pretty janky, so have wrapped a `Fade` transition around it

Reply:
I think it's better to just replace paper component props with
```
      componentsProps={{
        paper: {
          sx: {
            backgroundColor: theme => theme.palette.background.toolbar,
            minHeight: height ? `${height}` : 'auto',
          }
        }
      }}
```

https://github.com/mui/material-ui/issues/31073

Reply:
Good find! removing both of the components works nicely.